plugins/tmux-autoattach/tmux-autoattach.plugin.sh

```diff
@@ -16,14 +16,14 @@
 # [1] https://github.com/ohmybash/oh-my-bash/pull/332
 
 function _osh_plugin_tmux_autoattach_exit {
-  [[ ! $TMUX ]] && tmux -2u new -As0 && exit
+  [[ ! ${TMUX-} ]] && tmux -2u new -As0 && exit
 }
 
 function _osh_plugin_tmux_autoattach_detach {
-  [[ ! $TMUX ]] && tmux -2u new -As0
+  [[ ! ${TMUX-} ]] && tmux -2u new -As0
 }
 
-case $OSH_PLUGIN_TMUX_AUTOATTACH_BEHAVIOR in
+case ${OSH_PLUGIN_TMUX_AUTOATTACH_BEHAVIOR-} in
```

Comment:
This is unnecessary because `OSH_PLUGIN_TMUX_AUTOATTACH_BEHAVIOR ` is ensured to exist in an earlier part of the same file. The same for `zellij-autoattach`.